image/src/custom-mobilenet.ts

```diff
@@ -205,6 +205,15 @@ export async function getTopKClasses(labels: string[], logits: tf.Tensor<tf.Rank
 
 
 export class CustomMobileNet {
+    /**
+     * the truncated mobilenet model we will train on top of
+     */
+    protected truncatedModel: tf.LayersModel;
+    /**
+     * the training model for transfer learning
+     */
+    protected trainingModel: tf.LayersModel;
```

Comment:
`CustomMobileNet` is meant to be when a custom model is loaded, where `TeachableMobileNet` is meant to be the one for training.
I think this `trainingModel` variable can just be in `TeachableMobileNet` and that class can override `dispose`

Reply:
Done.